action.yml

```diff
@@ -9,16 +9,22 @@ inputs:
   module_download_mode:
     description: '-mod=<mode> value'
     required: false
+  go-version:
+    description: 'go version'
+    required: false
+    default: '1.20.x'
```

Comment:
We switch to 1.21 for the default version https://github.com/upfluence/private-actions/commit/cb52a200d8c8f1f424b120aacffd5ddbb5f1aa12 :)

Reply:
Good catch